index.html

```diff
@@ -3500,18 +3499,129 @@ <h3>
+                </p>
+        
+                <dl>
+                  <dt><dfn>equivalentId</dfn></dt>
+                  <dd>
+                    The value of <code>equivalentId</code> MUST be a 
+                    <a data-cite="INFRA#lists">list</a> where each item in the list 
+                    is a DID conforming to the requirements in Section 3.1.
+                  </dd>
+                  <dd>
+                    The relationship is a statement that each <code><a>equivalentId</a></code> 
+                    DID value is logically equivalent to the <code>id</code> property DID 
+                    value and thus identifies the same DID subject.
+                  </dd>
+                  <dd>
+                    Each <code><a>equivalentId</a></code> DID value MUST be produced by, and a form of, 
+                    the same <a>DID Method</a> as the <code>id</code> property DID value. 
+                    (e.g. <code>did:example:abc</code> == <code>did:example:ABC</code>)
+                  </dd>
+                  <dd>
+                    A conforming <a>DID Method</a> specification MUST guarantee that each 
+                    <code><a>equivalentId</a></code> DID value is logically equivalent to the 
+                    <code>id</code> property DID value.
+                  </dd>
+                  <dd>
+                    A resolving party MUST retain the DIDs from the <code>id</code> and 
+                    <code><a>equivalentId</a></code> properties to ensure any subsequent 
+                    interactions with any of the DIDs they contain are correctly handled 
+                    as logically equivalent DIDs (e.g. retain all variants in a database so an 
+                    interaction with any one maps to the same underlying account).
+                  </dd>
+                </dl>
+        
+                <div class="note" title="Equivalence and equivalentId">
+                  <p>
+                    <code><a>equivalentId</a></code> is a much stronger form of equivalence than 
+                    <code><a>alsoKnownAs</a></code> because the equivalence MUST be guaranteed by 
+                    the governing DID method. <code><a>equivalentId</a></code> represents a full 
+                    graph merge because the same DID document describes both the 
+                    <code><a>equivalentId</a></code> DID and the <code>id</code> property DID.
+                  </p>
+                </div>
+        
+              </section>
+      
+            <section>
+              <h4>canonicalId</h4>
+                <p>
+                  The <code><a>canonicalId</a></code> property is identical to the 
+                  <code><a>equivalentId</a></code> property except: a) it accepts only a single 
+                  DID as its value, and b) that DID is defined to be the canonical DID 
+                  for the DID subject within the scope of the containing DID document. 
```

Comment:
```suggestion
                  The <code><a>canonicalId</a></code> property is identical to the 
                  <code><a>equivalentId</a></code> property except: a) it accepts only a single 
                  value rather than a list, and b) that DID is defined to be the canonical ID 
                  for the DID subject within the scope of the containing DID document. 
```

Reply:
I wonder if it would be clearer for the portion of the paragraph that details the relationship canonicalId has to the equivalentId to instead be a note?